### ISSUE TITLE
Map all points, clustering them

### ISSUE THREAD

**Issue by waldoj** [openva/vabusinesses.org#26]
Like <a href="https://stackoverflow.com/questions/17371039/how-to-add-markers-bulk-in-leaflet">such</a>.

**Comment by waldoj** [comment]
This requires [a Leaflet plugin](https://github.com/Leaflet/Leaflet.markercluster) that requires Bower, but I can't install Bower because `Error: failed to fetch from registry: bower` whatever that means. Also, Bower is a package manager that you install with npm, a package manager, and you install that with apt-get, a package manager.

**Comment by waldoj** [comment]
I figured out the Bower problem, got the Leaflet plugin installed (I think), and started getting this added. But then I realized that I wasn't quite sure what the circumstance was under which I wanted it to run. I'm pretty sure that we want to cluster points whenever more than X results are being displayed at one time, at least at first. Maybe a next step will be that, whenever more than Y results are found in a search (when Y is a very large number), we take a random sample of Z% of those points and cluster-map those.
